conans/client/cmake.py

```diff
@@ -232,16 +307,26 @@ def build(self, conan_file, args=None, build_dir=None, target=None):
             args_to_string(args)
         ])
         command = "cmake --build %s" % arg_list
-        conan_file.run(command)
-
-    def test(self, conan_file, args=None, build_dir=None, target=None):
+        self._conanfile.run(command)
+
+    def _test_old(self, conan_file, args=None, build_dir=None, target=None):
```

Comment:
test wasn't released yet, so, is it really necessary to create old compatibility?